src/uu/du/src/du.rs

```diff
@@ -670,15 +668,16 @@ Try '{} --help' for more information.",
                             let secs = {
                                 match matches.value_of(options::TIME) {
                                     Some(s) => match s {
-                                        "accessed" => stat.accessed,
-                                        "created" => stat.created,
-                                        "modified" => stat.modified,
+                                        "ctime" | "status" => stat.modified,
+                                        "access" | "atime" | "use" => stat.accessed,
                                         _ => {
                                             show_error!(
-                                                "invalid argument 'modified' for '--time'
-    Valid arguments are:
-      - 'accessed', 'created', 'modified'
-    Try '{} --help' for more information.",
+                                                "invalid argument ‘{}’ for ‘--time’
```

Comment:
hmm, it isn't possible to use https://docs.rs/clap/2.31.0/clap/struct.Arg.html#method.possible_values to avoid that?

Reply:
Done.

Reply:
I think this block isn't necessary anymore.
clap will reject unknown argument

Reply:
Updated